core/message-tags-3.3.md

```diff
@@ -135,9 +135,14 @@ Servers MUST reply with the `ERR_INPUTTOOLONG` (`417`) error numeric if a client
 ## Security considerations
 
 Client-only tags should be treated as untrusted data. They can contain any value
-and are not validated by servers in any way.
+and are not validated by servers in any way. The server MAY normalise values by applying the unescape
+and escape process.
 
-Some specifications may involve servers accepting client-initiated tags without the client-only prefix. Such tag values SHOULD be validated by the server before being sent with any response to a client, unless they are specifically specified to be untrusted data.
+Some specifications may involve servers accepting client-initiated tags without the client-only prefix,
+they SHOULD define a validation process to be performed by the server.
+
+Tags without the client-only prefix or that fail validation MUST be removed by the server before being
+sent with any response to another client.
```

Comment:
I can see what's meant here, but I think you could more clearly write it as something like this:

```
Tags without the client-only prefix that are unexpected or otherwise fail validation MUST be removed by the server before being sent with any response to another client.
```

As currently written, it feels like it's trying to tell servers to remove all tags without the client-only prefix in all cases, where I think above is more what's meant to be conveyed?